lib/ex_doc/formatter/html.ex

```diff
@@ -128,7 +128,14 @@ defmodule ExDoc.Formatter.HTML do
 
   def ast_to_html({tag, attrs, ast}) do
     attrs = Enum.map(attrs, fn {key, val} -> " #{key}=\"#{val}\"" end)
-    ["<#{tag}", attrs, ">", ast_to_html(ast), "</#{tag}>"]
+
+    case tag do
```

Comment:
Are those changes related?

Reply:
They are. Otherwise the text inside `<pre>` will be printed in one long line